contextvh/src/main/java/contextvh/ContextEntity.java

```diff
@@ -30,6 +36,7 @@
 	 */
 	public ContextEntity(final EntityListener env, final String intendedStakeholder, final Integer slotID) {
 		super(env, intendedStakeholder, slotID);
+		customActions.addAction(perceptsFilter);
```

Comment:
It may be better if the action was gotten from the ActionContainer in the constructor, and the action added in the constructor for ActionContainer.